packages/utilities/psammead-locales/moment/ig.js

```diff
@@ -0,0 +1,51 @@
+const moment = require('moment');
+
+moment.defineLocale('ig', {
+  months: 'Jenụwarị_Febụwarị_Maachị_Eprel_Mee_Juun_Julaị_Ọgọọst_Septemba_Ọktọba_Nọvemba_Disemba'.split(
+    '_',
+  ),
+  monthsShort: 'Jen_Feb_Maa_Epr_Mee_Juu_Jul_Ọgọ_Sep_Ọkt_Nov_Dis'.split('_'),
+  weekdays: 'Sọnde_Mọnde_Tuzde_Wenesde_Tọsde_Fraịde_Satọde'.split('_'),
+  weekdaysShort: 'Sọn_Mọn_Tuz_We_Tọs_Frai_Sat'.split('_'),
+  weekdaysMin: 'Su_Mọ_Tu_We_Tọ_Fr_Sa'.split('_'),
+  longDateFormat: {
+    LT: 'HH:mm',
+    LTS: 'HH:mm:ss',
+    L: 'DD/MM/YYYY',
+    LL: 'D MMMM YYYY',
+    LLL: 'D MMMM YYYY HH:mm',
+    LLLL: 'dddd, D MMMM YYYY HH:mm',
```

Comment:
I believe this is as follows
```suggestion
    LLLL: 'dddd, MMMM D YYYY HH:mm',
```